src/static/helpers/slasHelper.ts

```diff
@@ -426,6 +450,9 @@ export async function loginRegisteredUserB2C(
     redirectURI: string;
     usid?: string;
     dnt?: boolean;
+  } & CustomQueryParameters,
+  options?: {
+    body?: CustomRequestBody;
```

Comment:
We should only expect body to just custom params since the required body is constructed via parameters or clientConfig that does not require users input when calling this function